lib/Stampie/Mailer.php

```diff
@@ -2,7 +2,13 @@
 
 namespace Stampie;
 
-use Stampie\Adapter\AdapterInterface;
+use GuzzleHttp\Psr7\MultipartStream;
```

Comment:
Neither do I. There is a PR that will create a MultipartStream factory that will abstract away this dependency as well. I'll do my best to make sure it get merged soon. 

Ref: https://github.com/php-http/message-factory/pull/30